<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransferClientClass</class>
 <widget class="QWidget" name="FileTransferClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileTransferClient</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="connectBtn">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipLEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入服务器ip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="connectStatusLabel">
       <property name="text">
        <string>连接状态：未连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openFolderBtn">
       <property name="text">
        <string>打开文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selPathBtn">
       <property name="text">
        <string>保存路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="selPath"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="historyInfo"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileTransferClient.qrc"/>
 </resources>
 <connections/>
</ui>
